coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelDataTypeToAvroType.java

```diff
@@ -110,9 +111,13 @@ private static Schema basicSqlTypeToAvroType(BasicSqlType relDataType) {
       case NULL:
         return Schema.create(Schema.Type.NULL);
       case TIMESTAMP:
-        Schema schema = Schema.create(Schema.Type.LONG);
+        schema = Schema.create(Schema.Type.LONG);
         schema.addProp("logicalType", "timestamp-millis");
         return schema;
+      case DATE:
```

Comment:
can we also add an inline comment here documenting the reason for this transformation?

Reply:
Added